docs/operations/query-concurrency.md

```diff
@@ -0,0 +1,174 @@
+# Laning strategy
+druid.query.scheduler.laning.strategy=hilo
+druid.query.scheduler.laning.maxLowPercent=20
+```
+
+See [Query prioritization and laning](../configuration/index.md#query-prioritization-and-laning) for details on query laning and the available query laning strategies.
+
+## Historical tiering
+
+Configure segment loading and tiers for Historicals as described in this section. In the examples below, we set up two tiers—hot and cold—for the Historicals and for the Brokers. We’ll instruct the Brokers to serve hot queries before cold queries. Short-running queries will be routed to the hot tiers, and long-running queries will be routed to the cold tiers.
+
+It is possible to separate Historical processes into tiers without having separate Broker tiers. This way, you can assign data from specific time intervals to specific tiers in order to support higher concurrency on hot data. 
+
+### Configure segment loading
+
+Druid stores data in segment files. Define a [load rule](rule-configuration.md#load-rules) to indicate how segment replicas should be assigned to different Historical tiers. For example, you may store segments of more recent data on more powerful hardware for better performance.
+
+There are three types of load rules: forever, interval, and period. Select the load rule that matches your use case for each Historical, whether you want all segments to be loaded, segments within a certain time interval, or segments within a certain time period. 
+
+In the load rule, define tiers in the `tieredReplicants` property. Provide descriptive names for your tiers, and specify how many replicas each tier should have. You can designate a higher number of replicas for the hot tier to increase the concurrency for processing queries.
+
+Example load rule with two Historical tiers, named “hot” and “\_default\_tier”:
+
+```
+{
+  "type" : "loadByPeriod",
+  "period" : "P1M",
+  "includeFuture" : true,
+  "tieredReplicants": {
+    "hot": 3,
+    "_default_tier" : 1
+  }
+}
+```
+
+See [Load rules](rule-configuration.md#load-rules) for more information on segment load rules. Visit [Tutorial: Configuring data retention](../tutorials/tutorial-retention.md) for an example of setting retention rules from the Druid web console.
+
+### Assign Historicals to tiers
+
+Assign the Historical to tiers by labeling the tier name and setting the priority value in the  `historical/runtime.properties` files.
+
+Example config for a Historical in the hot tier:
+
+```
+druid.server.tier=hot
+druid.server.priority=1
+```
+
+Example config for a Historical in the cold tier:
+
+```
+druid.server.tier=_default_tier
+druid.server.priority=0
+```
+
+See [Historical general configuration](../configuration/index.md#historical-general-configuration) for more details on these properties.
+
+## Broker tiering
+
+To set up Broker tiering, assign Brokers to tiers and configure query routing by the Router. You must set up Historical tiering before you can use Broker tiering.
```

Comment:
Why would I do Broker tiering? What is the added benefit. Consider stating that it's dependent on Historical tiering first.

Reply:
added content to top of "Service tiering" section